packages/page-accounts/src/Accounts/Account.tsx

```diff
@@ -131,8 +131,12 @@ function Account ({ account: { address, meta }, className = '', delegation, filt
 
   useEffect((): void => {
     if (balancesAll) {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
+      const bonded = stakingInfo?.stakingLedger.active || (stakingInfo?.stakingLedger as unknown as any)?.activeRing;
```

Comment:
Really don't want `activeRing` in here - it is not part of the Substrate base modules. (aka it is non-maintainable)